src/App.tsx

```diff
@@ -1,37 +1,50 @@
 import { useState } from "react";
 import "./styles/App.css";
 import SurvivorSelection from "./components/SurvivorSelection";
-import { Survivor, UserSelection } from "./utilities/types";
+import { Items, Survivor, UserSelection } from "./utilities/types";
 import Instructions from "./components/Instructions";
 import ItemSelection from "./components/ItemSelection";
 
 //app level: rendering the individual steps of the flow and nothing else
 function App() {
   const [userSelection, setUserSelection] = useState<UserSelection>(
-    {} as UserSelection
+    {userID: 0,
+    userSurvivor: {} as Survivor,
+    userItems: []
+  }
   ); //*to store survivor selected by user, user id, and items selected by user
 
+  const [numberOfItemsSelected, setNumberOfItemsSelected] = useState<number>(0)
+
   console.log(userSelection);
 
   const handleSurvivorSelection = (survivor: Survivor) => {
     setUserSelection((prevUserSelection) => ({
       ...prevUserSelection,
-      id: Date.now(),
+      userID: Date.now(),
       userSurvivor: survivor
     }));
   };
 
+  const handleItemSelection = (item: Items) => {
+    setUserSelection((prevUserSelection) => ({
+      ...prevUserSelection,
+      userItems: [...prevUserSelection.userItems, item]
+    }))
+    setNumberOfItemsSelected(prev => prev + 1)
+  };
+
   return (
     <>
-      {!userSelection.userSurvivor ? (
+      {userSelection.userID === 0 ? (
```

Comment:
this feels less intuitive, and promotes tribal knowledge > intuitive knowledge

---

- tribal: i just happen to know that `userID` is set as the current date once a survivor is selected (even though that's not very obvious) -> thus if a `userID` is set, a survivor must have been selected
- intuitive: there does not exist a survivor under `userSelection`, so don't show `<SurvivorSelection> etc